content/web-frameworks/orchard-core-cms/setup-orchard-core-cms.md

```diff
@@ -107,3 +109,23 @@ dotnet run
 
 # Example of admin page
 ![](images/orchard-core-cms-admin-page.png)
+
+# All Orchard Core Cms Template
+
+You can use `dotnet new` for listing all installed templates, and you can select by using `Short Name` of the template: 
+
+```
+Templates                                     Short Name      Language    Tags
+--------------------------------------------  --------------  ----------  ----------------------
+Orchard Core Cms Module                       ocmodulecms     [C#]        Web/Orchard Core/CMS
+Orchard Core Cms Web App                      occms           [C#]        Web/Orchard Core/CMS
+Orchard Core Theme                            octheme         [C#]        Web/Orchard Core/CMS
+Orchard Core Mvc Module                       ocmodulemvc     [C#]        Web/Orchard Core/Mvc
+Orchard Core Mvc Web App                      ocmvc           [C#]        Web/Orchard Core/Mvc
+```
+
+For exmaple:
+Orchard Core Cms Web App: 
```

Comment:
Could you please have a shot description to explain what "dotnet new occms" does?
For example,
`Orchard Core Cms Web App:` to create a new Orchard Core CMS website project.

Reply:
Sure